### PR TITLE
SG-30578 - Session Launcher Updates (DO NOT MERGE)

### PR DESCRIPTION
**Pull request by shaynacohen**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/5feb147acfb7e23bd6f78537a74b2b0fb83351b4/index.html)

File: docs/en/guides/pipeline-integrations/development/sgtk-developer-bootstrapping.md [hunk 1]
```diff
@@ -81,8 +81,25 @@ sgtk.LogManager().global_debug = True
 ## Part 3: Authentication
 
 When running a script that uses the Toolkit API outside of an environment where {% include product %} Toolkit has already been started, you will always need to authenticate.
+
 So before you can perform the bootstrapping, you need to authenticate the Toolkit API with your {% include product %} site.
 
+Upon launching Desktop and applications, you will be prompted to sign in to your {% include product %} site (example: `https://mysite.shotgrid.autodesk.com`). 
+
+![sign in](./images/sa-integrations-user-guide-session-launcher-01.png)
```
Comment (julien-lang) [inline docs/en/guides/pipeline-integrations/development/sgtk-developer-bootstrapping.md:89]: The image is quite large there and the icon is pixelized. We might want to improve that.

http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/5feb147acfb7e23bd6f78537a74b2b0fb83351b4/d587be80/?title=Integrations+User+Guide

File: docs/en/guides/pipeline-integrations/getting-started/advanced_config.md [hunk 2]
```diff
@@ -63,14 +63,28 @@ In this guide, you will use the Wizard in {% include product %} Desktop to gener
...
 ![New project](./images/advanced_config/2_new_project.png)
 
-**Step 2:** Launch the {% include product %} Desktop app and log in using the username and password from your {% include product %} site.
+**Step 2:** Launch the {% include product %} Desktop app and log in using the {% include product %} Application Session Launcher.
 
-![Login](./images/advanced_config/3_login.png)
+Upon launching Desktop and applications, you will be prompted to sign in to your {% include product %} site (example: `https://mysite.shotgrid.autodesk.com`). 
 
-![Load project](./images/advanced_config/4_load_project.png)
+![sign in](./images/sa-integrations-user-guide-session-launcher-01.png)
```
Comment (julien-lang) [inline docs/en/guides/pipeline-integrations/getting-started/advanced_config.md:70]: These images don't show up http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/5feb147acfb7e23bd6f78537a74b2b0fb83351b4/5d83a936/?title=Configuration+Setup

**Review by julien-lang — CHANGES_REQUESTED** [review]

File: docs/en/guides/pipeline-integrations/development/sgtk-developer-bootstrapping.md (hunk 1 above)
Comment (shaynacohen) [reply docs/en/guides/pipeline-integrations/development/sgtk-developer-bootstrapping.md:89]: Thanks @julien-lang ! Let's see how the images look in this next build.

File: docs/en/guides/pipeline-integrations/getting-started/advanced_config.md (hunk 2 above)
Comment (shaynacohen) [reply docs/en/guides/pipeline-integrations/getting-started/advanced_config.md]: Thanks @julien-lang ! Let's see how the images look in this next build.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/db0e6e71e0d7926edac6c0c3c032f1fc02dba9af/index.html)

**Comment by shaynacohen** [comment]
@julien-lang , how are these updates?
- http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/db0e6e71e0d7926edac6c0c3c032f1fc02dba9af/d587be80/?title=Integrations+User+Guide#logging-in-and-authenticating
- http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/db0e6e71e0d7926edac6c0c3c032f1fc02dba9af/3d8cc69a/?title=Bootstrapping+and+running+an+app#part-3-authentication
- http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/db0e6e71e0d7926edac6c0c3c032f1fc02dba9af/5d83a936/?title=Configuration+Setup
- http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/db0e6e71e0d7926edac6c0c3c032f1fc02dba9af/724152ce/

**Review by julien-lang — APPROVED** [review]
LGTM!

Please don't merge this PR just yet. It needs to be synchronized.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/db0e6e71e0d7926edac6c0c3c032f1fc02dba9af/index.html)

**Comment by shaynacohen** [comment]
@julien-lang can this be merged?

**Comment by julien-lang** [comment]
> @julien-lang can this be merged?

@shaynacohen No, not yet, unfortunately. Hopefully soon. I will let you know. Thanks.

**Comment by shaynacohen** [comment]
@julien-lang closing this PR since we have new docs. I've added you to the other PRs that we will move forward with.

**Pull request closed without merging**